pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md

```diff
@@ -7,17 +7,37 @@ formats:
   - ads
 ---
 
-Triggering CSS animations [relies on adding and removing classes](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations), done via JavaScript. You can achieve the same behavior on AMP pages by using the [`toggleClass`](/content/amp-dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.md#-all-elements) action. With `toggleClass`, AMP pages can add, remove, and toggle CSS classes like non-AMP pages.
+CSS animations enable web elements to transition from one CSS style configuration to another. The browser can start defined animations on load, but event triggered CSS animations [rely on adding and removing classes](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations). AMP supports both animation types.
+
+Use CSS when you have a smaller, contained animation that doesn't need to be precisely timed.
 
 ## Defining CSS and keyframes
 
 You can define CSS in AMP in the following ways:
 
+[filter="websites, stories"]
```

Comment:
syntax error, needs to be: `[filter formats="websites, stories"]`

http://localhost:8080/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#filtering-sections

Reply:
omg lol